src/idiom301.fuzion

```diff
@@ -0,0 +1,3 @@
+fib(n i32) i32 =>
+  if n<2 then n
+  else        (fib (n-1)) + (fib (n-2))
```

Comment:
line feed missing at EOF.
in vscode you can use option "files.insertFinalNewline": true
then this is done automagically

Reply:
🙈
thanks 👍️